src/components/UI/Button/Button.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Button.scss';
+
+const Button = ({
+  name,
+  clickHandler,
+  className,
+  alignRight,
+}) => (
+  <button
+    onClick={clickHandler}
```

Comment:
OPTIONAL you can call clickHandler onClick instead, so when you use your Button it is more native button alike.

Reply:
ok